Dockerfile.deploy

```diff
@@ -55,7 +64,13 @@ RUN pip install --no-cache-dir --exists-action=w --no-deps -r requirements/syste
     && pip install --no-cache-dir --exists-action=w --no-deps -r requirements/prod_without_hash.txt\
     && pip install --no-cache-dir --exists-action=w --no-deps -e .
 
-RUN echo -e "from olympia.lib.settings_base import *\n\
+# Link /usr/sbin/uwsgi and /usr/bin/uwsgi to deal with migration from Centos -> Debian
```

Comment:
I'm not quite sure what's the reason for this symlink. Is it because something outside of the Dockerfile that needs it? Maybe a brief description of that for future reference.

Reply:
Centos and Debian packages have uwsgi in different locations, until we clean up every ref in puppet I want to have both supported.

Reply:
`/usr/sbin/uwsgi` is the assumed location (by Puppet) about what command to run. To avoid breaking other things in Puppet (as I don't know the blast radius) I'm leaving the Puppet-assumed location of uWSGI unchanged and making sure the Docker image has a the file in the right place.